arm/models/models.py

```diff
@@ -491,3 +493,138 @@ def get_d(self):
             if '_sa_instance_state' not in key:
                 return_dict[str(key)] = str(value)
         return return_dict
+
+
+class SystemInfo(db.Model):
```

Comment:
The rule should be one class per file, but we can push that refactor to another time

Reply:
agree, the file at the moment is rather large and cumbersome 
if that is the general rule I can reactor with any other changes I make against v2_devel

Reply:
It's not formalized yet, so you're good here. A refactor that big needs to be it's own PR